app/models/annotation/nml/NmlResults.scala

```diff
@@ -0,0 +1,67 @@
+package models.annotation.nml
+
+import com.scalableminds.webknossos.datastore.SkeletonTracing.SkeletonTracing
+import com.scalableminds.webknossos.datastore.VolumeTracing.VolumeTracing
+import com.scalableminds.util.tools.Fox
+import com.typesafe.scalalogging.LazyLogging
+import play.api.libs.Files.TemporaryFile
+
+import scala.concurrent.ExecutionContext
+
+
+object NmlResults extends LazyLogging {
```

Comment:
The types are moved here from NmlService, as that one is now also an injected class